src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py

```diff
@@ -0,0 +1,66 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# AZURE CLI EventHub - NAMESPACE TEST DEFINITIONS
+
+import time
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+
+class EHNamespaceCURDScenarioTest(ScenarioTest):
+    from azure_devtools.scenario_tests import AllowLargeResponse
+
+    @AllowLargeResponse()
+    def test_eh_cluster(self):
+        self.kwargs.update({
+            'loc': 'southcentralus',
+            'rg': 'rgehcluster-cli-test',
+            'clustername': self.create_random_name(prefix='eventhubs-clus1-', length=20),
+            'namespacename': self.create_random_name(prefix='eventhubs-nscli', length=20),
+            'tags': {'tag1=value1'},
+            'tags2': {'tag2=value2'},
+            'capacity': 1,
+            'sku': 'Standard',
+            'tier': 'Standard'
+        })
+
+        # create Resource group
+        self.cmd('az group create --resource-group {rg} --location {loc}')
```

Comment:
use ResourceGroupPreparer() to create resource group as  https://github.com/Azure/azure-cli/blob/daa9d5d6807a63abba5c7a7e82e98645422ef9a1/src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py#L1146